e2e.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
```

Comment:
:+1:

Reply:
@kbeker At the same time it runs all the tests with appropriate configuration for behave itself (e.g. `--no-capture` for debugging) , so it is directly used to perform the tests the way we intended.